app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&send_courses=false&study_type%5B%5D=full_time&study_type%5B%5D=part_time&subjects%5B%5D=18&visa_status=false">Search for Italian teacher training courses</a>.
+
+## Japanese 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=languages&button=#funding-widget">Tax-free bursaries of £25,000 are available for all eligible trainee Japanese teachers</a>.
+
+As a Japanese teacher you'll help pupils learn a new language and open them up to a different culture.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&send_courses=false&study_type%5B%5D=full_time&study_type%5B%5D=part_time&subjects%5B%5D=19&visa_status=false">Search for Japanese teacher training courses</a>.
+
+## Latin
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=languages&button=#funding-widget">Tax-free bursaries of £25,000 are available for all eligible trainee Latin teachers</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&send_courses=false&study_type%5B%5D=full_time&study_type%5B%5D=part_time&subjects%5B%5D=A0&visa_status=false">Search for Latin teacher training courses</a>.
+
+## Mandarin
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=languages&button=#funding-widget">Tax-free bursaries of £25,000 are available for all eligible trainee Mandarin teachers</a>.
+
+As a Mandarin teacher you'll help pupils learn a new language and open them up to a different culture.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&send_courses=false&study_type%5B%5D=full_time&study_type%5B%5D=part_time&subjects%5B%5D=20&visa_status=false">Search for Mandarin teacher training courses</a>.
+
+## Mathematics 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=maths&button=#funding-widget">Tax-free scholarships of £30,000 and bursaries of £28,000 are available for eligible trainee maths teachers</a>.
```

Comment:
@caitlin-reidy same as previous comments re moving line of bursaries to after intro text.

Reply:
@caitlin-reidy Realise we've got the maths page but I still think we should have a line of intro for each of the subjects we have pages for. So they all follow the same format and are consistent. And also because people may not have read them and we want to get them excited about chemistry so they want to then read it.

Otherwise for this one we just have 3 links.